examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/demo_io.c

```diff
@@ -342,7 +362,7 @@ int8_t io_read_humidity( float* read_value )
  * @param  dec_prec the decimal precision to be used
  * @retval None
  */
-void floatToInt( float in, int32_t* out_int, int32_t* out_dec, int32_t dec_prec )
+static void floatToInt( float in, int32_t* out_int, int32_t* out_dec, int32_t dec_prec )
```

Comment:
:+1: 
Just one small note here - I wouldn't call it a `floatToInt` cause what it does it just splits the float number into two pieces. So maybe a `floatSplit` or something like that ? `FloatToInt` suggests a cast so it's misleading - at least for a game programmer :)